src/challenges.js

```diff
@@ -1,49 +1,123 @@
 // Desafio 1
-function compareTrue() {
-  // seu código aqui
+function compareTrue(param1, param2) {
+  return param1 && param2;
 }
 
 // Desafio 2
-function calcArea() {
-  // seu código aqui
+function calcArea(base, height) {
+  return (base * height) / 2;
 }
 
 // Desafio 3
-function splitSentence() {
-  // seu código aqui
+function splitSentence(sentence) {
+  return sentence.split(' ');
 }
 
 // Desafio 4
-function concatName() {
-  // seu código aqui
+function concatName(names) {
+  return `${names[names.length - 1]}, ${names[0]}`;
```

Comment:
O que você acha de deixar esta função um pouco mais legível?
```suggestion
  let namesLastWord = names[names.length - 1];
  let namesFirstWord = names[0];
  return `${namesLastWord}, ${namesFirstWord}`;
```